src/components/Search/Search.tsx

```diff
@@ -9,16 +9,16 @@ import VibeComponentProps from "../../types/VibeComponentProps";
 import CloseIcon from "../Icon/Icons/components/CloseSmall";
 import SearchIcon from "../Icon/Icons/components/Search";
 import { NOOP } from "../../utils/function-utils";
-import { IconType } from "../Icon/IconConstants";
+import { Icon } from "src/types/SubIcon";
```

Comment:
[Optional]
We have three different 'conventions' for import type:
```
import { Icon } from "src/types/SubIcon";
import { SubIcon } from "../../types/SubIcon";
import { SubIcon } from "src/types/SubIcon";
```
Probably we can choose one...